static/src/javascripts/bootstraps/enhanced/football.js

```diff
@@ -0,0 +1,426 @@
+                if (b) {
+                    bean.fire(b, 'click');
+                }
+            },
+            () => {
+                extras.forEach(extra => {
+                    if (extra && extra.content && extra.importance) {
+                        addComponent(extra.content, extra.importance);
+                    }
+                });
+            }
+        );
+    }
+};
+
+const renderTable = (
+    competition: string,
+    extras: Extras,
+    template: string
+): void => {
+    extras[2] = {
+        ready: false,
+    };
+
+    $.create(
+        `<div class="js-football-table"
+              data-link-name="football-table-embed"></div>
+        `
+    ).each(container => {
+        tableFor(competition)
+            .fetch(container)
+            .then(() => {
+                if ($('.table__container', container).length > 0) {
+                    extras[2] = {
+                        name: 'Table',
+                        importance: 3,
+                        content: container,
+                        ready: true,
+                    };
+                } else {
+                    extras[2] = undefined;
+                }
+
+                renderExtras(extras, template);
+            })
+            .catch(() => {
+                delete extras[2];
+                renderExtras(extras, template);
+            });
+    });
+};
+
+const loading = (
+    elem: HTMLElement,
+    message: string = 'Loading…',
+    link: {
+        text: string,
+        href: string,
+    }
+): void => {
+    bonzo(elem).append(
+        bonzo.create(`
+            <div class="loading">
+                <div class="loading__message">${message}</div>
+                    <a href="${link.href}"
+                       class="loading__link">${link.text}</a>
+                    <div class="loading__animation"></div>
+            </div>
+        `)
+    );
+};
+
+const loaded = (elem: HTMLElement): void => {
+    $('.loading', elem).remove();
+};
+
+const init = (): void => {
+    // We're doing this as to have one redraw
+    const extras = [];
+
+    let dropdownTemplate;
+
+    isMatch((match: Object): void => {
+        extras[0] = {
+            ready: false,
+        };
+        if (match.pageType === 'stats') {
+            renderNav(match);
+        } else {
+            const $h = $('.js-score');
+            const scoreBoard = new ScoreBoard({
+                pageType: match.pageType,
+                parent: $h,
+                responseDataKey: 'matchSummary',
+                autoupdated: match.isLive,
+            });
+
+            renderNav(match, (resp, $nav): void => {
+                dropdownTemplate = resp.dropdown;
+
+                // Test if template is not composed of just whitspace. A content validation check, apparently.
+                if (scoreBoard.template && !/^\s+$/.test(scoreBoard.template)) {
+                    scoreBoard.loadFromJson(resp.matchSummary);
```

Comment:
We've lost a bit of code here. Was that intentional?
```js
scoreBoard.endpoint = endpoint;
```

Reply:
Yes. I checked `scoreBoard` and endpoint is not used anywhere ...